net-p2p/eiskaltdcpp/metadata.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>klondike@gentoo.org</email>
+		<name>Francisco Blas (klondike) Izquierdo Riera</name>
+	</maintainer>
 	<longdescription lang="en">
 	EiskaltDC++ is a multi-platform program that uses the Direct Connect and
 	ADC protocol. It is compatible with other DC clients, such as the original DC
```

Comment:
@klondi 
Please update program description to something like this:
```
EiskaltDC++ is a cross-platform program that uses the Direct Connect and
Advanced Direct Connect protocols. It is compatible with DC++, AirDC++,
FlylinkDC++ and other DC clients. EiskaltDC++ also interoperates with all
common DC hub software.
```
It comes from [README](https://github.com/eiskaltdcpp/eiskaltdcpp#description) and is used in packages for Debian and other distros.

Reply:
Updated this, thanks tehnick!